src/main/java/teammates/common/util/Const.java

```diff
@@ -964,7 +964,7 @@ private Const() {
         public static final String COURSE_STATS = "/course/stats";
         public static final String INSTRUCTOR_COURSE_DETAILS = "/courses/details";
         public static final String INSTRUCTOR_COURSE_DETAILS_DELETE_ALL_STUDENTS = "/courses/details/deleteAllStudents";
-        public static final String INSTRUCTOR_COURSE_DETAILS_ALL_STUDENTS_CSV = "/courses/details/allStudentsCsv";
+        public static final String INSTRUCTOR_COURSE_DETAILS_ALL_STUDENTS_CSV = "/students/csv";
```

Comment:
Follow the convention. 
```suggestion
        public static final String STUDENTS_CSV = "/students/csv";
```